src/Components/design/Accordion/AccordionTable/AccordionTable.css

```
@@ -28,12 +29,28 @@
   text-align: right;
 }
 
+.accordion-table__cell--message {
+  display: flex;
+  align-items: left;
+  justify-content: flex-start;
+}
+
 .accordion-table__row-header--first .accordion-table__cell {
   font-size: var(--font-size-small);
   font-weight: var(--font-weight-bold);
   color: var(--contrast-70);
+  /* text-align: left; */
+}
+
+.accordion-table__deadline__cell {
+  padding-left: 38px;
 }
 
 .accordion-table__row-panel {
   padding: 4px 0;
 }
+
+.accordion-table__spacer {
+  grid-column: none;
+  width: 1em;
```

I'd recommend using rems instead of ems. (rems are always relative to root font size `<html>`  instead of the parent's)
```suggestion
  width: 1rem;
```